src/main/webapp/app/exam/manage/exams/exam-detail.component.html

```diff
@@ -78,44 +74,126 @@ <h3>
+                                    </li>
+                                </ul>
                             </td>
                             <td class="d-none d-md-table-cell">
-                                <a *ngIf="isAtLeastInstructor" [routerLink]="['exercise-groups']" class="btn btn-primary" id="exercises-button-groups_table">
-                                    <fa-icon [icon]="'list-alt'"></fa-icon><span>{{ 'artemisApp.examManagement.exerciseGroups' | translate }}</span>
+                                <a *ngIf="isAtLeastInstructor" [routerLink]="getExamRoutesByIdentifier('test-runs')" class="btn btn-info">
+                                    <fa-icon [icon]="'user'"></fa-icon>
+                                    <span>{{ 'artemisApp.examManagement.testRun.testRun' | translate }}</span>
                                 </a>
                             </td>
                         </tr>
                         <tr>
                             <td class="d-none d-md-table-cell">3</td>
                             <td class="d-none d-md-table-cell">
-                                <span>{{ 'artemisApp.examManagement.checklist.tableItem.testRun' | translate }}</span>
-                            </td>
-                            <td class="d-none d-md-table-cell">
-                                <span>{{ 'artemisApp.examManagement.checklist.descriptionItem.testRun' | translate }}</span>
+                                <span>{{ 'artemisApp.examManagement.checklist.tableItem.register' | translate }}</span>
                             </td>
                             <td class="d-none d-md-table-cell">
-                                <a *ngIf="isAtLeastInstructor" [routerLink]="['test-runs']" class="btn btn-info">
+                                <span>{{ 'artemisApp.examManagement.checklist.descriptionItem.register' | translate }}</span>
+                                <hr />
+                                <ul class="list-unstyled">
+                                    <li class="list-group-item border-1">
+                                        <span>
+                                            <jhi-exam-checklist-check
+                                                [checkAttribute]="exam.numberOfRegisteredUsers && exam?.numberOfRegisteredUsers > 0"
+                                            ></jhi-exam-checklist-check>
+                                            <span>{{ 'artemisApp.examManagement.checklist.textitems.leastonestudent' | translate }}</span>
+                                        </span>
+                                    </li>
+                                    <li class="list-group-item border-1">
+                                        <span>{{ 'artemisApp.examManagement.checklist.textitems.studentsregistered' | translate }}</span>
+                                        <span> {{ exam.registeredUsers?.length }} </span>
```

Comment:
Set the default to 0 here?

```suggestion
                                        <span> {{ exam.registeredUsers?.length || 0 }} </span>
```